### ISSUE TITLE
blank spaces in the bottom

### ISSUE THREAD

**Issue by EspadaRunica** [rozniak/xfce-winxp-tc#157]
Brother, 2 things.
- The first is that I don't understand why it looks like this after having installed everything, is there a solution? (Attached image)
![asd](https://user-images.githubusercontent.com/44514442/190531178-ed4c8849-6cf3-4c14-b479-1ec75637438b.png)

- The second is that I did an autoinstall of this in bash (tested on Linux Mint).
I'm not very good at writing bash yet xD, but it works to install all the packages contained in your project. once finished you have to customize the desktop manually, maybe it will help you or maybe I can extract something that works for you.
[winxp-tc.sh.zip](https://github.com/rozniak/xfce-winxp-tc/files/9579714/winxp-tc.sh.zip)
Thank you for such a beautiful project.

**Comment by rozniak** [comment]
> The first is that I don't understand why it looks like this after having installed everything, is there a solution? (Attached image)

Yes - you need to install the forked XFCE panel under `/submodules`, it contains a fix for this backported to 4.16.

> The second is that I made a bash installer (tested on Linux Mint 21 "Vanessa").

Could have a look - I'm not sure when I'll start working on an installer atm. Needs to be distro-agnostic and whatnot - I have a lot of notes in my head I should write down on #81 .

**Comment by EspadaRunica** [comment]
Then maybe it can help you extract the dependencies, it's for all ubuntu based :P

**Comment by EspadaRunica** [comment]
The problem with the gray background was solved, I only have that space left, that with other themes the menu opens well.
![2022-09-16_09-38](https://user-images.githubusercontent.com/44514442/190640756-29a95de6-5971-48cf-9334-ee418d75674c.png)

**Comment by rozniak** [comment]
> Then maybe it can help you extract the dependencies, it's for all ubuntu based :P

Yeah I'll have a look it might come in handy for 'post-install' tasks. I've been kind of brainstorming how the project can try to support whatever distro the user is running (and how to handle various setups - if they have no DE installed yet or if they are not on XFCE). It's kind of complicated but I should write some thoughts down.

> The problem with the gray background was solved, I only have that space left, that with other themes (like adwaita) the menu opens well.

Weird... does this happen if you adjust the default system font in XFCE settings to Tahoma Regular 8pt?

**Comment by EspadaRunica** [comment]
take it easy, don't stress and do it to be happy, as a hobby, otherwise you'll end up hating the project xD
If I knew how to program I would help you, but what I know is very basic. Maybe if you need to test and don't have time, I can help.
Telegram [chmodmasx](https://t.me/chmodmasx)

**Comment by rozniak** [comment]
Yea don't worry I 'chip away' rather than work on this flat out. It's just good to keep bugs in mind so I don't forget to sort them out eventually. :grin:
